marketplace-core/src/main/java/org/pentaho/marketplace/endpoints/MarketplaceService.java

```diff
@@ -75,7 +76,7 @@ public StringOperationResultDTO hello() {
   @GET
   @Path( "/plugins" )
   @Produces( MediaType.APPLICATION_JSON )
-  public IterablePluginOperationResultDTO getPlugins() {
+  public Response getPlugins() {
```

Comment:
Ok. I initially did that, but the project don't have javax.servlet dependency. So I tryed to keep in the scope of JAX-RS. I'll add the dependency, then.